frontend/src/containers/RecipePage/Recipe.css

```diff
@@ -0,0 +1,154 @@
+.overall-container {
+  margin: 0px auto;
+  padding: 0px;
+}
+
+.user-edit-button {
+  display: inline-block;
+  float: right;
+  margin: 20px 26px;
+  border: 3px solid #ffa616;
+  border-radius: 13%;
+  color: #ffa616;
+  font-family: 'Nunito', sans-serif;
+  font-size: 14px;
+  height: 4em;
+  padding-left: .5em;
+  padding-right: .8em;
+  line-height: 3em;
+  font-weight: 600;
+  width: 120px;
+  text-align: center;
+  margin-top: 70px;
+}
+
+.recipe-icon {
+  margin-top: 50px;
+  height: 80px;
+  line-height: 80px;
+  text-align: left;
+  font-size: 23.4px;
+  color: white;
+  padding: 0px;
+}
+
+.recipe-photo {
+  width: 90%;
+  height: 500px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.tags-bar {
+  display: flex;
+  justify-content: center;
+
+}
+
+#recipe-descrip {
+  text-align: center;
+  padding: 1% 3% 3% 3%;
+
+}
+
+#hr {
+  border: 0.5px solid lightgrey;
+  width: 90%;
+  margin: 0 auto;
+}
+
+#ingredient-box {
+  display: flex;
+  justify-content: center;
+  margin-top: 1%;
+  padding: 1% 3% 3% 3%;
+  margin-bottom: -1%;
+}
+
+#ingredient-ul {
+  column-count: 2;
+  column-gap: 50px;
+  padding: 6% 3% 3% 3%;
+  width: 100%;
+
+}
+
+#recipe-title, #direction-title, #ingredient-title {
+  margin-top: 2%;
+  text-align: center;
+  padding: 1% 3% 3% 3%;
+  font-size: 50px;
+  font-weight: bold;
+}
+
+#direction-container {
+  margin-bottom: 20px;
+  text-align: left;
+  padding: 1% 3% 3% 3%;
+}
+
+.single-direction {
+  color: black;
```

Comment:
As with most of the text on the site, I think the design doc indicates this is actually a dark grey. The headings seem to be black, while the snippet, the instructions, and the directions are all that dark grey used elsewhere.

Reply:
changed color to dark grey

Reply:
Maybe we should delete this unused color change?